apps/volk_option_helpers.cc

```diff
@@ -225,15 +226,8 @@ void option_list::help()
             help_line += this_option->shortform + " [ " + this_option->longform + " ]";
         }
 
-        switch (help_line.size() / 8) {
-        case 0:
-            help_line += "\t";
-        case 1:
-            help_line += "\t";
-        case 2:
-            help_line += "\t";
-        case 3:
-            help_line += "\t";
```

Comment:
This code is just meant to line up the tables that are displayed by `volk_profile --help` and `volk_config-info --help`. The new version gets rid of the fall-through warnings, and is simpler.